manifests/repo/download_postgresql_org.pp

```diff
@@ -16,15 +16,21 @@
     $label1 = 'redhat'
     $label2 = 'rhel'
   }
+  $default_baseurl = "https://download.postgresql.org/pub/repos/yum/${postgresql::repo::version}/${label1}/${label2}-\$releasever-\$basearch"
 
-  yumrepo { 'yum.postgresql.org':
+  $baseurl_real = pick($postgresql::repo::baseurl,$default_baseurl,'undefined')
+  if ( $baseurl_real == 'undefined' ){
```

Comment:
Nit pick really, but there has to be a better value to match against, maybe `undef` itself (https://docs.puppet.com/puppet/latest/reference/lang_data_undef.html)? Merely that a string feels like a bad signal to use.